pkg/config/config.go

```
@@ -116,10 +114,6 @@ func KyvernoServiceAccountName() string {
 	return kyvernoServiceAccountName
 }
 
-func KyvernoDeploymentName() string {
```

Why changing the deployment name only? I find these envs quite useful as they can be accessed globally.

---

Accessing them globally is useful but if we forgot to set the env var we don't see it. Shall we panic if the env var is missing ?

---

That's not a bad idea. I say we stick with the env and panic when it's not set.